tmt/steps/provision/testcloud.py

```diff
@@ -381,7 +381,7 @@ def prepare_ssh_key(self, key_type: Optional[str] = None) -> None:
         if self.key:
             self.debug("Extract public key from the provided private one.")
             command = Command("ssh-keygen", "-f", str(self.key[0]), "-y")
-            public_key = self.run(command).stdout
+            public_key = self._run_guest_command(command).stdout
```

Comment:
I'm not sure this should change to `_run_guest_command`. I don't think that keygen generation should be subject to verbosity levels. IMO it is purely for 'debug' 

Similar to other keygen command on line 393

Reply:
`silent=True` then? Tried that in 8edbbfc0